estimation/matrices.R

```diff
@@ -119,10 +121,9 @@ matrices$cost_transit_other_2016 = matrices$cost_transit_other_2018
 matrices = unpick(matrices, cost_transit_monthly_2018)
 
 # Walking
-matrices$length_pedestrian_2016 = 0L
-matrices$length_pedestrian_2018 = 0L
-matrices$ttime_pedestrian_2016 = 0L
-matrices$ttime_pedestrian_2018 = 0L
+matrices$ttime_pedestrian_2018 = matrices$length_pedestrian_2018 / 5.0 * 60
```

Comment:
Consider naming "magic numbers & formulas" so that outsider would also have better understanding why the division is done. f.ex:

``` 
insert_name_here_why_we_divide = 5.0 * 60
matrices$ttime_pedestrian_2018 = matrices$length_pedestrian_2018/insert_name_here_why_we_divide 
```

Reply:
Thanks!